ionic_user_interface/src/views/Help.vue

```diff
@@ -38,19 +38,19 @@
 
         <hr class="my-4" id="section2" />
         <p class="lead my-4">1. Upload the photo of a climbing wall</p>
-        <img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_1.jpg`" />
+        <ion-img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_1.jpg`" />
         <br />
         <p class="lead my-4">2. The holds will be marked automatically</p>
-        <img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_2.jpg`" />
+        <ion-img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_2.jpg`" />
         <br />
         <p class="lead my-4">3. Create a custom route by following the prompts</p>
-        <img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_3.jpg`" />
+        <ion-img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_3.jpg`" />
         <br />
         <p class="lead my-4">4. Post to share with other gym goers</p>
-        <img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_4.jpg`" />
+        <ion-img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_4.jpg`" />
         <br />
         <p class="lead my-4">5. Vote, grade and comment on other's routes</p>
-        <img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_5.jpg`" />
+        <ion-img class="screenshot" :src="`${publicPath}assets/screenshots/Screenshot_5.jpg`" />
```

Comment:
The picture quality is still bad. Chrome problem. I can try to fix if you don't have any fixes in mind

Reply:
Ok, you can try to fix